rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section1/rule_1_5.json

```diff
@@ -0,0 +1,787 @@
+{
+    "rule-1-5-a": {
+        "Section": 1,
+        "Rule": 5,
+        "Test": "a",
+        "test_description": "A one building segment, single zone building has proposed on site renewable energy production. The relationship between PCI, PBPnre, PBP, and BBP agrees with PCI + ((PBPnre - PBP)/BBP)-0.05 <= PCIt. RESULT: pass",
+        "expected_rule_outcome": "pass",
+        "standard": {
+            "rule_id": "1-5",
+            "ruleset_reference": "4.2.1.2",
+            "rule_description": "When on-site renewable energy generation exceeds the thresholds defined in Section 4.2.1.1, the methodology defined in this section shall be used to calculate the PCIt. ",
+            "applicable_rmr": "Proposed Model",
+            "rule_assertion": "=",
+            "comparison_value": "Baseline Model",
+            "rule_dependency": 0,
+            "mandatory_rule": "Yes",
+            "schema_version": "0.0.34"
+        },
+        "rmr_transformations": {
+            "proposed": {
+                "id": "ASHRAE229 1",
+                "weather": {
+                    "climate_zone": "CZ4A"
+                },
+                "calendar": {
+                    "is_leap_year": false
+                },
+                "data_timestamp": "2024-02-12T12:00Z",
+                "ruleset_model_descriptions": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "floor_name": "Floor 1",
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
+                                                        "floor_area": 1393.5456,
+                                                        "lighting_space_type": "OFFICE_OPEN_PLAN"
+                                                    }
+                                                ]
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ],
+                        "output": {
+                            "id": "Output 1",
+                            "performance_cost_index": 1.5,
+                            "total_proposed_building_energy_cost_including_renewable_energy": 2,
```

Comment:
There must be ONE unique value of the ```total_proposed_building_energy_cost_including_renewable_energy``` in the case (a). However, there are 2 in lines 58 and 105.

Can you modify/delete one of these?

Reply:
Can you see the following?

- Case (a):  The ```(pci_set[0] + (pbp_nre_set[0] - pbp_set[0]) / bbp_set[0])``` value is 2.5 which **shouldn't** be greater than the ```pci_target_set``` value (2.45) to **pass**.
-  Case (b): The ```(pci_set[0] + (pbp_nre_set[0] - pbp_set[0]) / bbp_set[0])``` value is 2.5 which **should** be greater than the ```pci_target_set``` value (2.45) to **fail**.
- Case (g): The ```performance_cost_index_target``` value must exist, but it's missing.

Reply:
Juan My apologies. Please disregard my previous comment. I made a small mistake in the rule_check equation.

After correcting, case (b) passed unexpectedly. 

The ```(pci_set[0] + ((pbp_nre_set[0] - pbp_set[0]) / bbp_set[0]))``` value is 2.5 and the ```pci_target_set``` value is 2.7. To make this case fail, we should lower the ```pci_target_set``` value than 2.45.